source/trackers.json

```diff
@@ -1072,6 +1072,7 @@
         "adjust.net.in": "adjust",
         "adjust.world": "adjust",
         "apptrace.com": "adjust",
+        "adobedc.net": "adobe_experience_cloud",
```

Comment:
`adobe_experience_cloud` has `site_analytics` category, `adobedc.net` belongs to ads

Reply:
> adobedc.net

It seems analytics/personalization
https://experienceleague.adobe.com/docs/experience-platform/edge-network-server-api/overview.html?lang=en

Reply:
yep, `adobe_experience_league` is better, I think.